sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Generated/Models/DigitalTwinsDescription.cs

```diff
@@ -51,11 +53,13 @@ public DigitalTwinsDescription()
         /// 'Suspending', 'Restoring', 'Moving'</param>
         /// <param name="hostName">Api endpoint to work with
         /// DigitalTwinsInstance.</param>
+        /// <param name="privateEndpointConnections">The private endpoint
+        /// connections.</param>
         /// <param name="publicNetworkAccess">Public network access for the
         /// DigitalTwinsInstance. Possible values include: 'Enabled',
         /// 'Disabled'</param>
-        public DigitalTwinsDescription(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), DigitalTwinsIdentity identity = default(DigitalTwinsIdentity), System.DateTime? createdTime = default(System.DateTime?), System.DateTime? lastUpdatedTime = default(System.DateTime?), string provisioningState = default(string), string hostName = default(string), IList<PrivateEndpointConnection> privateEndpointConnections = default(IList<PrivateEndpointConnection>), string publicNetworkAccess = default(string))
-            : base(location, id, name, type, tags, identity)
+        public DigitalTwinsDescription(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), DigitalTwinsIdentity identity = default(DigitalTwinsIdentity), SystemData systemData = default(SystemData), System.DateTime? createdTime = default(System.DateTime?), System.DateTime? lastUpdatedTime = default(System.DateTime?), string provisioningState = default(string), string hostName = default(string), IList<PrivateEndpointConnection> privateEndpointConnections = default(IList<PrivateEndpointConnection>), string publicNetworkAccess = default(string))
```

Comment:
This is a breaking change, since a new parameter is inserted, instead of appended to the end.

Reply:
Thanks! This was not intended to be a breaking change, I will attempt to rectify with a manual code change. SystemData is not writeable by the SDK consumer.

Reply:
I've added a constructor that has the optional parameter at the end. Let me know if that works.